src/spring/azext_spring/tests/latest/test_asa_app_scenario.py

```diff
@@ -219,7 +219,10 @@ def test_app_crud_1(self, resource_group, spring, app):
         ])
 
     @SpringResourceGroupPreparer(dev_setting_name=SpringTestEnvironmentEnum.ENTERPRISE_WITH_TANZU['resource_group_name'])
-    @SpringPreparer(**SpringTestEnvironmentEnum.ENTERPRISE_WITH_TANZU['spring'])
+    @SpringPreparer(dev_setting_name="AZURE_CLI_TEST_DEV_SPRING_NAME_TANZU",
```

Comment:
Do we have to hard code this?

Reply:
This's a customization to add parameter '--enable-config-server' since there's cmd to bind app to the config server. The default settings of ENTERPRISE_WITH_TANZU does not contain the `--enable-config-server`.